pages/verification/index.tsx

```diff
@@ -99,12 +108,12 @@ export default function Verification(props: PageProps) {
 
         const anonTokenFromCookie = Cookies.get(ITokenEnum.anonymousToken)
         if(anonTokenFromCookie) {
-            sendAnonymousMessage(anonTokenFromCookie)
+            sendAnonymousMessage(anonTokenFromCookie, {
+                replaceRoute: true,
+                withLoading: true
+            })
         }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are logical and seem to be handling the loading state and route replacement correctly based on the new optional parameters introduced in the `sendAnonymousMessage` function. However, there is no null check for the `anonTokenFromCookie`. If a null or undefined value is passed, it could lead to potential issues when setting the 'Authorization' header.
```diff
         const anonTokenFromCookie = Cookies.get(ITokenEnum.anonymousToken)
+        if (!anonTokenFromCookie) {
+            console.error('Anonymous token from cookie is missing');
+            return;
+        }
         if(anonTokenFromCookie) {
             sendAnonymousMessage(anonTokenFromCookie, {
                 replaceRoute: true,
                 withLoading: true
             })
         }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->